code-challenges.js

```diff
@@ -2,22 +2,50 @@
 
 // --------------------1) Create a function that takes in a string and returns a coded message. The coded message converts 'a' to 4, 'e' to 3, 'i' to 1, and 'o' to 0.
 
-var secretCodeWord1 = "lackadaisical"
+var secretCodeWord1 = "lakadaisical"
 // Expected output: "l4ck4d41s1c4l"
 var secretCodeWord2 = "gobbledygook"
 // Expected output: "g0bbl3dyg00k"
 
+//check each character and convert their value to a number if they are either a,e,i,o
 
-
+const convertCode = (str) => {
+ let splitString = str.split("");
+ 
+ let letters = splitString.map(val => { 
+     if (val.includes("a")) {
+         val = 4
+     } 
+     else if (val.includes("e")) {
+         val = 3
+     } 
+     else if(val.includes("i")) {
+         val = 1
+     } 
+     else if (val.includes("o")) {
```

Comment:
is there a reason why you used .includes instead of `===`? just because using .includes on a single character is a little strange